src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -402,9 +403,15 @@ def check_read_nwb(self, nwbfile_path: str):
 
         if recording.get_num_segments() == 1:
             # Spikeinterface behavior is to load the electrode table channel_name property as a channel_id
+
+            if not self.is_lfp_interface:
+                electrical_series_path = f"acquisition/{electrical_series_name}"
+            else:
+                electrical_series_path = f"processing/ecephys/LFP/{electrical_series_name}"
```

Comment:
```suggestion
            if self.is_lfp_interface:
                electrical_series_path = f"processing/ecephys/LFP/{electrical_series_name}"
            else:
                electrical_series_path = f"acquisition/{electrical_series_name}"
                
```

Reply:
this feels a little more natural than `if not / else`

Reply:
Yes, you are right. I was going for more common path first but readability trumps that I feel.